### PR TITLE
Feature/Checkbox ListItem DAO-615

### PR DESCRIPTION
**Pull request by RakeshUP**

### CODE REVIEW

File: packages/ui-components/src/components/checkbox/checkboxListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  isMulti?: boolean;
+};
+
+export const CheckboxListItem: React.FC<CheckboxListItemProps> = ({
+  label,
+  helptext,
+  multiSelect,
+  isActive = false,
+  disabled = false,
+  isMulti = false,
```
Comment (Fabricevladimir) [inline packages/ui-components/src/components/checkbox/checkboxListItem.tsx]: Can you explain a bit what this prop is for? It's a bit ambiguous especially with the other prop `multiselect`

File: packages/ui-components/src/components/checkbox/checkboxListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    default: <IconCheckboxDefault />,
+  },
+  radio: {
+    active: <IconRadioSelected />,
+    multi: <IconRadioDefault />,
+    default: <IconRadioDefault />,
+  },
+};
+
+export type CheckboxListItemProps = {
```
Comment (Fabricevladimir) [inline packages/ui-components/src/components/checkbox/checkboxListItem.tsx:24]: How do we toggle the state of the checkbox?

**Review by Fabricevladimir — COMMENTED** [review]
Good work, and thanks for fixing the circular dependency issue. Left a couple of questions.

File: packages/ui-components/src/components/checkbox/checkboxListItem.tsx (hunk 1 above)
Comment (sepehr2github) [inline packages/ui-components/src/components/checkbox/checkboxListItem.tsx]: There is no onClick or onChange implemented so when I click on the list item nothing happened!. And I think we should have a better way of handling the props. So based on the design we have two types (multi-selection: yes or no) and we have three states for multi-selection (default/active/multi) and two for single selection (default/active) all status would change by user clicks

File: packages/ui-components/src/components/checkbox/checkboxListItem.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  const iconType = isActive ? 'active' : isMulti ? 'multi' : 'default';
+
+  return (
+    <Container
+      data-testid="checkboxListItem"
+      isActive={isActive}
+      disabled={disabled}
+      isMulti={isMulti}
+    >
+      <HStack isActive={isActive} disabled={disabled} isMulti={isMulti}>
```
Comment (sepehr2github) [inline packages/ui-components/src/components/checkbox/checkboxListItem.tsx]: and also can you add cursor: pointer for hover?

**Review by sepehr2github — COMMENTED** [review]
Thank you! Rakash. I left a few comments for you. 👍

**Comment by RakeshUP** [comment]
@Fabricevladimir @sepehr2github I've added onclick prop and a controlled way to toggle the state inside the component itself. I've changed the other ambiguous props too

**Review by Fabricevladimir — APPROVED** [review]
Looks great! Thanks

**Review by sepehr2github — APPROVED** [review]

**Pull request merged**